src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "Empty",
+    "package": "{4f3766dd-5201-42c0-aa83-41e71f56c7a0}",
+    "id": "1011"
+  },
+  "description": {
+    "text": "A project template for creating an ASP.NET Core SignalR real-time application with example SignalR Hubs.",
+    "package": "{4f3766dd-5201-42c0-aa83-41e71f56c7a0}",
+    "id": "1012"
+  },
+  "order": 100,
+  "icon": "vs-2017.3/Empty.png",
```

Comment:
Does SignalR have an Icon?

Reply:
SignalR does not have an official icon AFAIK. @davidfowl?

Reply:
It used to but maybe we use the service icon?

Reply:
SignalR itself has no logo as it is not a "product". The Azure SignalR Service has a logo but we probably shouldn't use that here. The best case scenario here IMO would be to get a new logo for the template that illustrates the real-time scenario (something like the API template but maybe with more... arrows?).

That of course requires us to have design resources we're willing to spend on this template :). Until then, I think the empty or API logos would be good candidates:

Empty: ![Empty template logo](https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/Web.ProjectTemplates/content/EmptyWeb-CSharp/.template.config/vs-2017.3/Empty.png?raw=true)

API: ![API template logo](https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/.template.config/vs-2017.3/WebAPI.png?raw=true)

Reply:
i can grab the service icon no problem.

Reply:
grabbed it from here: https://docs.microsoft.com/en-us/azure/media/index/azure-signalr.svg

Reply:
I'm not sure we should use the service icon.

Reply:
i'm kind of with you on that but i've added it for now. we can test and see how much we hate it?

Reply:
Ask a PM Manager what to do. Using the service icon is likely wrong.